elasticsearch/tests/test.sh

```diff
@@ -14,10 +22,12 @@ if [ "${SKIPBUILD}" -eq 0 ]; then
   build
   source results/last_build.env
   hab pkg install --binlink --force "results/${pkg_artifact}"
-  hab pkg install --binlink --force core/which
-  hab pkg install --binlink --force core/jre8
+  hab svc load "${pkg_ident}"
   popd > /dev/null
   set +e
+
+  # Give some time for the service to start up
+  sleep 10
```

Comment:
change to `sleep 30` so it builds consistently.

Reply:
Fixed